<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>downloadTools</class>
 <widget class="QWidget" name="downloadTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>下载工具</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>301</height>
    </rect>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>651</width>
      <height>301</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Start"/>
    <widget class="QWidget" name="NewDownload">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>622</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="Layout">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="DownloadLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTextEdit" name="URL">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9.07563pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9.07563pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="placeholderText">
            <string>请在此黏贴下载路径！</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="FileInfoLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="FileInfoWidget">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="rowCount">
            <number>0</number>
           </property>
           <property name="columnCount">
            <number>3</number>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column/>
           <column/>
           <column/>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>618</width>
             <height>108</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="FolderLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QComboBox" name="DisplayFolder">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="OpenFolder">
             <property name="minimumSize">
              <size>
               <width>45</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>45</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>浏览</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="Download">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>315</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>315</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>315</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>立即下载</string>
      </property>
     </widget>
     <zorder>layoutWidget</zorder>
     <zorder>Download</zorder>
     <zorder>comboBox</zorder>
     <zorder>OpenFolder</zorder>
    </widget>
    <widget class="QWidget" name="Downloading">
     <widget class="QTableWidget" name="downloadingTableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>631</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Downloaded">
     <widget class="QTableWidget" name="downloadedTableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>631</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTreeWidget" name="stateTreeWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>131</width>
      <height>191</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>状态</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>新建下载</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>正在下载</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>下载完成</string>
     </property>
    </item>
   </widget>
   <zorder>stateTreeWidget</zorder>
   <zorder>stackedWidget</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
